### DISCUSSION TITLE
Question.  Is it possible to configure the email context of an email via keycloakify?

### DISCUSSION THREAD

**Discussion by Tonypz** [keycloakify/keycloakify#620] [General]
I've seen the instructions, but it's not clear how you can create your own email template on react
( example https://docs.keycloakify.dev/email-customization)

maybe there is an example somewhere?

**Comment by garronej** [comment]
Hello @Tonypz,  

No, unfortunately it's not yet possible to work with React for creating a Keycloak theme.  

All keycloakify does at the moment is it let's you bundle your email theme in the JAR but you still have to work with FreeMarker.  

I'll be workign on this but it won't land soon.
